cgroup2/utils_test.go

```diff
@@ -41,6 +47,36 @@ func TestParseCgroupFromReader(t *testing.T) {
 	}
 }
 
+func TestParseStatCPUPSI(t *testing.T) {
+	const examplePSIData = `some avg10=1.71 avg60=2.36 avg300=2.57 total=230548833
+full avg10=1.00 avg60=1.01 avg300=1.00 total=157622356`
+
+	fakeCgroupDir := t.TempDir()
+	statPath := filepath.Join(fakeCgroupDir, "cpu.pressure")
+
+	if err := os.WriteFile(statPath, []byte(examplePSIData), 0o644); err != nil {
+		t.Fatal(err)
+	}
+
+	st := getStatPSIFromFile(filepath.Join(fakeCgroupDir, "cpu.pressure"))
+	if diff := cmp.Diff(st, stats.PSIStats{
```

Comment:
What is the benefit of using `go-cmp`? What about using `assert.Equal`?

Reply:
done